pageserver/src/layered_repository.rs

```diff
@@ -1326,17 +1347,19 @@ impl LayeredTimeline {
             // open layer to hold changes newer than the cutoff.) Then we write
             // the frozen layer to disk, and replace the in-memory frozen layer
             // with the new on-disk layers.
-            let FreezeLayers {
-                frozen,
-                open: maybe_new_open,
-            } = oldest_layer.freeze(last_record_lsn)?;
+            //
+            let freeze_layers = oldest_layer.freeze(last_record_lsn)?;
+            let frozen = freeze_layers.frozen;
+            let maybe_new_open = freeze_layers.open;
```

Comment:
Was this change intentional?

Reply:
It was at first but became unneeded after further refactoring.